SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -74,8 +74,8 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bEnableNetCullDistanceInterest(true)
 	, bEnableNetCullDistanceFrequency(false)
 	, FullFrequencyNetCullDistanceRatio(1.0f)
-	, bUseSecureClientConnection(false)
-	, bUseSecureServerConnection(false)
+	, bUseSecureClientConnectionInPackagedBuilds(false)
```

Comment:
This is a breaking change

Reply:
Oof. I'll revert and change the display name then to prevent that.